src/video-iframe-integration.js

```diff
@@ -185,7 +185,7 @@ export class AmpVideoIntegration {
     if (!(method in this.methods_)) {
       return;
     }
-    this.methods_[method].call();
+    this.methods_[method]();
```

Comment:
I'm having trouble verifying if this change is safe, or if we should be setting `this` to a value. I'll investigate before merging